examples/player/simid_player.js

```diff
@@ -131,7 +139,7 @@ class SimidPlayer {
    * Plays a SIMID  creative once it has responded to the initialize ad message.
    */
   playAd() {
-    this.contentVideoElement_.pause();
```

Comment:
I like the thought of moving this to where the other video stuff happens, but by doing that I think this means that we will wait until the SIMID creative is ready before pausing the content.
I think this could cause problems if there is a slow connection.  For example, say there is supposed to be a commercial break at 0:30. At 0:30, playAd would be called, but say it takes 2 seconds for the simid ad to download and be ready. The init message would not resolve until 0:32, meaning 2 seconds of post-ad-break content would play, before it got paused and the ad started.

Although, since with this example code the ad is manually initialized before you manually click playAd, maybe this isnt a problem? 

@ryanthompson591 thoughts?